README.md

```diff
@@ -0,0 +1,47 @@
+### Abstract:
+- This user-friendly web app allows users to record and archive their ideas. After giving their idea a name and description in the title and body input fields respectively, users can save their idea and it will appear on a card. Idea cards can be favorited or deleted. Adding to the functionality of this application, users can filter only their favorited ideas by clicking the button “Show Starred Ideas.”  The features included are:
+- The ability to save a new idea each time the page loads
+- The ability to delete any previously saved ideas
+- The ability to favorite idea cards when a user clicks the star button
+- The ability to filter only the favorited ideas
+
+### Installation Instructions:
+- Git clone this repo to your local machine
+- Cd into that directory
+- Enter the command open index.html to open the browser
+- Have fun recording your ideas!
+### Preview of App:
+![App preview image 1](assets/App_Preview_01.png)
+![App preview image 2](assets/App_Preview_02.png)
+
+### Context:
+- We spent 20-25 hours working on this project. We are in week four of Mod 1.
+
+### Contributors:
+https://github.com/LDeLaRosa13
```

Comment:
can you add line breaks in between each of these? 

> To create a line break or new line, end a line with two or more spaces, and then type return.

Reply:
Yes! Good call. I just added line breaks and pushed this branch again.